src/clr-icons/utils/shape-template-observer.ts

```diff
@@ -5,51 +5,43 @@
  */
 import { ShapeTemplateObservables } from '../interfaces/icon-interfaces';
 
-export const changeHandlerCallbacks: ShapeTemplateObservables = {};
-
-export class ShapeTemplateObserver {
-  private static singleInstance: ShapeTemplateObserver;
-
-  private callbacks: ShapeTemplateObservables = changeHandlerCallbacks;
+export namespace ShapeTemplateObserver {
```

Comment:
I'm not happy with the namespace names I gave. Let's change this to `export namespace ShapeTemplateObserverModule`